java/autoscaler/src/main/java/com/example/bigtable/autoscaler/Autoscaler.java

```diff
@@ -88,6 +88,11 @@
    */
   public static double CPU_PERCENT_TO_UPSCALE = .7;
 
+  /**
+   * Number of seconds to sleep after a scaling operation to wait for changes to take effect.
+   */
+  private static int SLEEP_DURATION_S = 600;
```

Comment:
The example is scheduled to run every minute.  Do we need to wait for 10 minutes instead?